projects/meteora-dammv2/index.js

```diff
@@ -0,0 +1,19 @@
+const { get } = require('../helper/http')
+
+const METEORA_API = 'https://dammv2-api.meteora.ag'
+
+// https://docs.meteora.ag/api-reference/damm-v2/overview
+async function tvl() {
+  const { data } = await get(`${METEORA_API}/pools/global-metrics`)
```

Comment:
please use on chain data to export token balances rather than an off chain USD value